guides/v2.3/release-notes/2.3.0-install.md

```diff
@@ -1,43 +1,48 @@
 ---
 
 ## Prerequisites 
 
 In addition to the [standard Magento prerequisites]({{ site.baseurl }}/guides/v2.3/install-gde/prereq/prereq-overview.html), make sure you have obtained and installed:
 
-* *Open Source Access*. The Open Source 2.3.0 Alpha Composer package is available to everyone with Magento authentication keys. If you do not have authentication keys, see [Get your authentication keys]({{page.baseurl}}/install-gde/prereq/connect-auth.html) for more information.
-* *Commerce Access*. The Commerce 2.3.0 Alpha Composer package is only available to Magento partners, as well as approved merchant developers and independent extension developers. In addition, access is granted only to those who complete and sign [Magento's Pre-Release Software Agreement](https://partners.magento.com/portal/pre-release-agreement). You will be notified of your approval to access the {{site.data.var.ee}} 2.3.0 Alpha repository within three business days. After you receive approval you will be able to install {{site.data.var.ee}} 2.3.0 Alpha using these installation instructions.
+* *{{site.data.var.ce}} Access*. The {{site.data.var.ce}} 2.3.0 Beta Composer package is available to everyone with Magento authentication keys. If you do not have authentication keys, see [Get your authentication keys]({{page.baseurl}}/install-gde/prereq/connect-auth.html) for more information.
+* *{{site.data.var.ee}} Access*. The {{site.data.var.ee}} 2.3.0 Beta Composer package is only available to Magento partners, as well as approved merchant developers and independent extension developers. In addition, access is granted only to those who complete and sign [Magento's Pre-Release Software Agreement](https://partners.magento.com/portal/pre-release-agreement). You will be notified of your approval to access the {{site.data.var.ee}} 2.3.0 Beta repository within three business days. After you receive approval you will be able to install {{site.data.var.ee}} 2.3.0 Beta using these installation instructions.
 * *PHP 7.1 or 7.2 and extensions*. See instructions here: [Install PHP 7.1 for Ubuntu]({{ site.baseurl }}/guides/v2.3/install-gde/prereq/php-ubuntu.html#instgde-prereq-php71-ubuntu) or [Install PHP 7.1 for Centos]({{ site.baseurl }}/guides/v2.3/install-gde/prereq/php-centos.html#php-centos-71).
 * *Composer*. Install using:
     ```bash
     curl -sS https://getcomposer.org/installer | sudo php -- --install-dir=/usr/bin --filename=composer
     ```
 
-## Install 2.3.0 Alpha
+## Install 2.3.0 Beta
 
-1. Install the Open Source or Commerce Alpha package.
+{: .bs-callout .bs-callout-info }
+To minimize storage issues, only the latest Alpha release and the last three Beta releases will be available for download from repo.magento.com. These instructions are for installing the Beta release. To install the latest Alpha release, replace `--stability=beta` with `--stability=alpha` in the {{site.data.var.ce}} and {{site.data.var.ee}} URLs that follow.
```

Comment:
This isn't correct; to install the latest Alpha while Beta packages are also available, you need to specify the exact alpha version (2.3.0-alpha79 is the last Alpha).  The --stability flag specifies minimum allowed stability, not which stability to install.  With the exact version specified, --stability should be removed completely, making the command "composer create-project --repository=https://repo.magento.com magento/project-community-edition=2.3.0-alpha79 <install_directory_name>"